src/components/study/display-study-status.tsx

```diff
@@ -57,7 +57,15 @@ const StatusBlock: React.FC<StatusLabels & { jobId?: string | null }> = ({ type,
                     <InfoComponent jobId={jobId} />
                 </Flex>
             ) : (
-                <Text>{label}</Text>
+                <Text
```

Comment:
This looks fine, the only suggestion I have is this component is getting a bit complicated and "fancy" with the logic. ie returning a ternary and then having the "else" condition on the ternary also doing some conditional logic on the `c` prop. I think this is fine for now, if we end up having to make any more changes or add any more conditions i'm just making a mental note that we should simplify the logic so that this component is more readable down the line